device-types/Dell/Unity-XT-380.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Dell
+model: Unity XT 380
+slug: dell-unity-xt-380
+part_number: 100-566-075-03
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
+interfaces:
+  - name: SP-A ETH0
+    type: 1000base-t
+    mgmt_only: true
+  - name: SP-B ETH0
+    type: 1000base-t
+    mgmt_only: true
+  - name: SP-A I/O Module 1 Eth Port 0
```

Comment:
By the looks these are storage controllers therefore, you need to added `device-bays` then create the controller as a separate device with the interfaces for that controller on.